src/views/conference/2018/index/l10n.json

```diff
@@ -10,7 +10,9 @@
     "conference-2018.location": "Where:",
 
     "conference-2018.desc1": "Join us for the Scratch@MIT conference, a playful gathering of educators, researchers, developers, and other members of the worldwide Scratch community.",
-    "conference-2018.desc2": "We're planning a very participatory conference, with an entire day of hands-on workshops and lots of opportunities for peer-to-peer discussion and collaboration.",
+    "conference-2018.desc2": "We're planning a very participatory conference, with an entire day of hands-on workshops and lots of opportunities for peer-to-peer discussion and collaboration. The conference is intended primarily for adults who support young people learning Scratch.
```

Comment:
You have to end it with a quotation mark and a comma

Reply:
Oh right. Thanks.